### ISSUE TITLE
COVID19 Immunity paper data

### ISSUE THREAD

**Issue by mikessh** [antigenomics/vdjdb-db#315]
Data from our collaborative project with Efimov lab, [SARS-CoV-2 epitopes are recognized by a public and diverse repertoire of human T-cell receptors
](https://www.cell.com/immunity/fulltext/S1074-7613(20)30469-6)

**Comment by wangshun1121** [comment]
Are TCR sequencing reads available of this paper? TCR repos can be found in VDJdb and iReceptor, and we want more details!

**Comment by ustervbo** [comment]
Dear wangshun1121,

Which details do you need?

The data for the  tetramer-positive TCRs is deposited here in `chunks/covid19-2020-05-19.txt`. The raw reads is accessible through the iReceptor gateway (https://gateway.ireceptor.org).

Best wishes,
Ulrik

**Comment by wangshun1121** [comment]
Dear Ulrik:

I have downloaded your data table from the iReceptor gateway, there are indeed more details. However, there are still more details we need. Clonotype frequency for example, in VDJdb all COVID associated TCR frequencies were released, and on iReceptor gateway, all TCR seq were shown but with no Clonotype frequency. 

You mentioned raw reads, I didn't find fastq reads accession. Or otherwise, tables on iReceptor are raw reads you mentioned?
